### DISCUSSION TITLE
[Feature] The ability to automatically, permanently cache some parts of the map for local, networkless, airgapped use

### DISCUSSION THREAD

**Discussion by ThatCoffeeGuy** [immich-app/immich#9767] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hey guys,

I was wondering if it's possible to permanently cache and save parts of the map (preferably where there are images) so that in case of a network issue or airgapped use, the map still could be browsed. I understand map tiles are HUGE, hence why I think it would work best with saving only the areas with photos.

I believe this effort would lower the dependency on external services. 

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
You can host a tile server in your local network (if you're up for that) and then configure a custom style.json URL that points at that server.

**Reply by ThatCoffeeGuy** [reply]
That's good to know, but the request is about this happening **automatically**. I agree it's not the most important or highest priority feature, but something I personally would love to see happening.
